INSTALL

```diff
@@ -68,8 +68,7 @@ you may need to install the development packages (look for a "-dev"
 postfix) in addition to the core packages.  
 
 python 2.7 or later (http://www.python.org/)
-    Iris requires Python 2.7 or later, but is not currently compatible with
-    Python 3.
+    Iris requires Python 2.7 or later.
```

Comment:
3.3 is left out because it's just old. 3.5 cannot yet be supported until Cartopy supports it.